src/components/DiffView/index.spec.tsx

```diff
@@ -43,4 +45,25 @@ describe(__filename, () => {
       expect(root.find(Diff).at(index)).toHaveProp('hunks', diff.hunks);
     });
   });
+
+  it('renders a header with diff stats', () => {
+    const root = render();
+
+    expect(root.find(`.${styles.header}`)).toHaveLength(1);
+    expect(root.find(`.${styles.stats}`)).toHaveLength(1);
+    expect(root.find(`.${styles.stats}`)).toIncludeText('+++ 2--- 0');
```

Comment:
This is good but it only covers one hunk. Can you add another test for a diff with more than one hunk?

Reply:
ah yes, thanks for that!